llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -1431,9 +1499,15 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
       ConvertedShadow = convertShadowToScalar(ConvertedShadow, IRB);
       Value *ConvertedShadow2 =
           IRB.CreateZExt(ConvertedShadow, IRB.getIntNTy(8 * (1 << SizeIndex)));
-      CallBase *CB = IRB.CreateCall(
-          Fn, {ConvertedShadow2,
-               MS.TrackOrigins && Origin ? Origin : (Value *)IRB.getInt32(0)});
+
+      SmallVector<Value *, 4> Args = {
```

Comment:
Should't this be SmallVector<Type *, 3>?

Reply:
It's a vector of `Value *` (these are Args, not ArgsTy). I've shrunk it to 3.